### ISSUE TITLE
video uploads?

### ISSUE THREAD

**Issue by lmccart** [lmccart/art-software#9]
* how to display
* alt text

**Comment by REAS** [comment]
We could start the *alpha* version with only image, but I feel video is important to best show the work of the artists. If the video has language in it, alt text should be required. With the other tutorials we're making to help artists with uploads, we can explain why and how for this? 

I think we should not include video for PDF (or other ebook formats).
